internal/service/employeeaccessgrant/resource.go

```diff
@@ -110,3 +90,24 @@ func (r *employeeAccessGrantRS) Delete(ctx context.Context, req resource.DeleteR
 
 func (r *employeeAccessGrantRS) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
 }
+
+func (r *employeeAccessGrantRS) createOrUpdate(ctx context.Context, tfModelFunc func(context.Context, any) diag.Diagnostics, diagnostics *diag.Diagnostics, state *tfsdk.State) {
```

Comment:
Nice re-use! If I understand correctly, this means the grant is idempotent and replaces the existing one.

Reply:
correct